src/pages/export-page/ExportPage.tsx

```diff
@@ -32,7 +32,13 @@ export const ExportPage = () => {
         developmentPlanEvent === TimetableEventKey.TimetableUpdated
     );
     return [
-      ...timetableEvents,
+      ...timetableEvents.map((x) => {
```

Comment:
can we use something meaningful rather than "x"? "Event" maybe?

Reply:
yep! this was me being lazy and forgot to change it!